### ISSUE TITLE
Allow org-admin to disable to_ids flag of synced events

### ISSUE THREAD

**Issue by cudeso** [MISP/MISP#5298]
### Expected behavior
Currently organisation admins can not change the to_ids flag of events received via a sync from other organizations. It's not always possible to do this via proposals.

Can the "disable to_ids flag" be added as an option in the user roles list?

**Comment by iglocska** [comment]
This is something we definitely want to avoid. Changing the IDS flag is tampering with the data - on a local instance level you can also use local tags as an alternative.

**Comment by cudeso** [comment]
Understood. 
But then you'll need something similar to an option in the pull rules to convert the to_ids flag to "tag:to_ids:true". It's possible to do this with PyMISP, but that's only when the event is already ingested, not during the sync.

**Comment by iglocska** [comment]
Indeed, I mean it's nothing formalised - use whatever tag makes sense for you (for example: "flagged as false positive") and then add it as a negative filter for your restSearch query (`"tags": {"NOT": ["flagged as false positive"]}`).

Just out of curiosity, where did you have trouble with the proposal override?

**Comment by cudeso** [comment]
Proposal override: 'human' delay ;-).

A useful option in the pull rules screen would be to add "add tag X" on new events and "add tag Y" on updates of existing events. That option is missing for the moment.

**Comment by RichieB2B** [comment]
What we do is run an ingestion MISP that selectively pushes events to a MISP used as a source for detection. The push can be automatic or manual defending on source, age, etc.
